src/v/security/audit/audit_log_manager.cc

```diff
@@ -421,6 +486,11 @@ ss::future<> audit_sink::stop() {
     co_await _gate.close();
 }
 
+ss::future<> audit_sink::update_auth_status(bool status) {
+    return _audit_mgr->container().invoke_on_all(
+      [status](audit_log_manager& mgr) { mgr._auth_misconfigured = status; });
+}
+
```

Comment:
for clarity sake, can this be a bool_class or something?  `status == true` is apparently 'bad' but that's not immediately obvious

Reply:
renaming to auth_misconfigured